client/containers/Configuration/Configuration.jsx

```diff
@@ -19,10 +19,38 @@ class Configuration extends Component {
 
     this.state = {
       activeTab: 1,
-      scripts: props.scripts || { }
+      scripts: props.scripts || { },
+      errors: { }
     };
   }
 
+  shouldComponentUpdate(nextProps) {
```

Comment:
This should just check if a component update is needed. It should not modify the state.